backend/src/transaction_event/protocol_level_tokens.rs

```diff
@@ -641,23 +653,58 @@ impl PreparedTokenUpdate {
         .execute(tx.as_mut())
         .await?;
 
+        self.update_metrics_plt_cumulative_event_count(tx, transaction_index).await?;
+
         match self.event_type {
             TokenUpdateEventType::Mint => {
                 if let Some(ref target) = self.target {
+                    let account_exists = self.check_if_account_exists(tx, target).await?;
+
                     self.update_total_minted(tx).await?;
                     self.update_account_balance(tx, target, &self.plt_amount_change).await?;
+                    if !account_exists {
+                        self.update_metrics_plt_unique_account_count(tx, transaction_index).await?;
```

Comment:
We never decrement this counter again?

Reply:
The balance of PLTs can become 0 again. PLTs are like smart contract tokens in that behavior. The transaction fee is paid in CCD (not a PLT token).

Your described behavior applies to CCD. It is in general hard to get a balance of 0 CCD on an account that had a positive balance before since you pay transaction fees in CCD. A small dust amount is usually left on an account since it is difficult to construct a transaction that transfers all your CCD away while having enough to pay for the tx fee.